### PR TITLE
TSDK-206 As a developer I can use the DAML broker seamlessly for testing my app (documentation)

### PR DESCRIPTION
**Pull request by mundacho**
## Purpose

The purpose of the PR is to setup CI/CD pipeline for documentation and publishing of a docusaurus website for the bifrost-daml-broker

## Approach

CI/CD pipeline was implemented using GH actions, website was deployed to github pages at https://topl.github.io/bifrost-daml-broker/

## Testing

The workflows was tested and a version of the website is available at https://topl.github.io/bifrost-daml-broker/.

## Tickets
* closes Topl/Brambl#86

### CODE REVIEW

**Comment by mundacho** [comment]
@mgrand-topl The 13,034 lines are just the SVG files and things that you normally should not see.

File: microsite/docusaurus.config.js [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+
+  themeConfig:
+    /** @type {import('@docusaurus/preset-classic').ThemeConfig} */
+    ({
+      colorMode: {
+        defaultMode: 'light',
+        disableSwitch: true,
+      },
+      navbar: {
+        title: 'Bifrost DAML Broker',
```
Comment (mgrand-topl) [inline microsite/docusaurus.config.js:56]: Bifrost ==> Topl

**Review by mgrand-topl — APPROVED** [review]

File: microsite/docs/intro.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+- A Java Virtual Machine (JVM).- The bifrost-daml-broker is a Java application, and thus Java is needed.
+- [Coursier](https://get-coursier.io/docs/cli-installation).- A simple command line tool (CLI) to
+to run Java applications without any setup. It is very easy to install.
+- [DAML SDK](https://docs.daml.com/getting-started/installation.html).- We need the DAML SDK to integrate with DAML
+- [Docker](https://www.docker.com/products/docker-desktop/).- We are using Docker to launch the Bifrost node easily. 
+
+### Broker Requirements
+
+To launch the broker you need to provide it with a `keyfile.json` file and the password for that file.
```
[the remark this answers is not in the document]
Comment (mundacho) [reply microsite/docs/intro.md:26]: A new issue has been created at https://topl.atlassian.net/browse/TSDK-260 to work on an easy to use utility for this UC.

File: microsite/docs/reference/architecture.md [hunk 3]
```diff
@@ -0,0 +1,39 @@
+---
+sidebar_position: 1
+---
+
+# Architecture
+
+We illustrate the architecture of an DAML application that integrates with the Bifrost node.
+
+![Architecture Diagram](img/damlIntegration.jpg)
```
[the remark this answers is not in the document]
Comment (mundacho) [reply microsite/docs/reference/architecture.md:9]: done

File: microsite/docs/reference/architecture.md [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+## Description of the Components
+
+### react-ui
+
+The react-ui is the UI of the application. The application provides this UI, which interacts with the JSON API usually through the `@react/ledger` Typescript library.
+
+### json-api
+
+The JSON API is a component provided by DAML. It is basically an executable JAR file that acts as an intermediary between a JSON API and the more complete Ledger gRPC API of the DAML Node. Applications can expose it through an nginx reverse proxy.
+
```
[the remark this answers is not in the document]
Comment (mundacho) [reply microsite/docs/reference/architecture.md:24]: done.

File: microsite/docusaurus.config.js (hunk 1 above)
Comment (mundacho) [reply microsite/docusaurus.config.js:56]: done

**Pull request merged**